aldryn_faq/tests/test_base.py

```diff
@@ -42,7 +48,28 @@ def assertCountEqual(a, b):
         return assertCountEqual(a, b)
 
 
-class AldrynFaqTestMixin(TestUtilityMixin, object):
+class CleanUpMixin(object):
+
+    def reload_urls(self):
```

Comment:
I would add this under the `TestUtilityMixin` instead of a new mixin class

Reply:
CleanUpMixin class removed and  added reload_urls function under TestUtilityMixin